lib/ts/recipe/thirdpartypasswordless/types.ts

```diff
@@ -495,6 +518,7 @@ export type APIInterface = {
         | undefined
         | ((input: {
               phoneNumber: string;
+              tenantId: string;
               options: PasswordlessAPIOptions;
               userContext: any;
           }) => Promise<
```

Comment:
TypeThirdPartyPasswordlessEmailDeliveryInput and TypeThirdPartyPasswordlessSmsDeliveryInput should take tenantId

Reply:
Same goes for in the passwordless / emailpassword / emailverification recipes

Reply:
Done